### PR TITLE
Update xhr-basic.html

### PR DESCRIPTION
**Pull request by webdev4422**
Change 'var' to 'const, let', so it relate to ajax-finish.html

### CODE REVIEW

File: javascript/apis/fetching-data/xhr-basic.html [hunk 1]
```diff
@@ -61,20 +61,20 @@ <h2>The Conqueror Worm, <em>Edgar Allen Poe, 1843</em></h2>
...
 
     <script>
-      var verseChoose = document.querySelector('select');
-      var poemDisplay = document.querySelector('pre');
+      const verseChoose = document.querySelector('select');
+      const poemDisplay = document.querySelector('pre');
 
       verseChoose.onchange = function() {
-        var verse = verseChoose.value;
+         verse = verseChoose.value;
```
Comment (chrisdavidmills) [inline javascript/apis/fetching-data/xhr-basic.html:68]: This needs a `const`

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi @mrxdkox ! Thanks for fixing this up.

I just had one comment for you to address, but apart from that we are good to go.

File: javascript/apis/fetching-data/xhr-basic.html (hunk 1 above)
Comment (mrxdkox) [reply javascript/apis/fetching-data/xhr-basic.html]: Got it, thx.

**Review by chrisdavidmills — APPROVED** [review]
Great, thanks!

**Pull request merged**